scripts/updateForBundling.js

```diff
@@ -0,0 +1,67 @@
+const fs = require('fs');
+
+// Function to update package.json
+function updatePackageJson() {
+  const packagePath = './package.json';
+
+  fs.readFile(packagePath, 'utf8', (err, data) => {
+    if (err) {
+      console.error(`Error reading package.json: ${err}`);
+      return;
+    }
+
+    try {
+      const packageJson = JSON.parse(data);
+
+      // Update package name if necessary
+      if (packageJson.name && packageJson.name === '@salesforce/core') {
+        packageJson.name = '@salesforce/core-bundle';
+      }
+
+      // Remove 'prepack' and 'prepare' scripts
+      if (packageJson.scripts) {
```

Comment:
These two fields are deleted because it will block the publishing process of esbuild.